commit 4029747c592cb2d59805b3a4e7a8963fcdcdbeb1
Author: John David Anglin <danglin@gcc.gnu.org>
Date:   Mon Sep 6 17:37:29 2021 +0000

    Update hppa libm-test-ulps
    
    (cherry picked from commit d8cf84ac7e504663dfeb2bb45d8d48ae81effe05)

diff --git a/sysdeps/hppa/fpu/libm-test-ulps b/sysdeps/hppa/fpu/libm-test-ulps
index 90e16a72692e9199..3d60fc25a14d053f 100644
--- a/sysdeps/hppa/fpu/libm-test-ulps
+++ b/sysdeps/hppa/fpu/libm-test-ulps
@@ -1104,7 +1104,7 @@ float: 8
 ldouble: 1
 
 Function: "tgamma_downward":
-double: 8
+double: 9
 float: 7
 
 Function: "tgamma_towardzero":
